packages/manager/modules/anthos/src/dashboard/host/order/anthos-dashboard-host-order.controller.js

```diff
@@ -0,0 +1,84 @@
+import {
+  MAX_ADDONS,
+  PRICE_DURATION,
+} from './anthos-dashboard-host-order.constants';
+import { extractHostAddonsFromAnthosCatalog } from './anthos-dashboard-host-order.utils';
+
+export default class AnthosDashboardHostOrderController {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    $timeout,
+    AnthosTenantsService,
+    User,
+    coreConfig,
+  ) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.$timeout = $timeout;
+    this.AnthosTenantsService = AnthosTenantsService;
+    this.User = User;
+
+    this.addons = [];
+    this.currencySymbol = coreConfig.getUser().currency.symbol;
+    this.expressOrderUrl = '';
+    this.isLoading = false;
+    this.canOrder = false;
+  }
+
+  $onInit() {
+    this.isLoading = true;
+
+    this.$q
+      .all({
+        anthosCatalog: this.AnthosTenantsService.getAnthosCatalog(),
+        expressOrderUrl: this.User.getUrlOf('express_order'),
+      })
+      .then(({ anthosCatalog, expressOrderUrl }) => {
+        this.addons = extractHostAddonsFromAnthosCatalog(anthosCatalog);
+        this.expressOrderUrl = expressOrderUrl;
+      })
+      .catch(() => {
+        const i18nError = 'anthos_dashboard_host_order_init_error';
+        const message = this.$translate.instant(i18nError);
+        this.displayAlerterMessage('error', message);
```

Comment:
**suggestion**: since the variables `i18nError` & `message` are used only once, we can omit the variables and directly pass them to the function.